fyde-access-proxy/install-aws.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: Install in AWS
+parent: Fyde Access Proxy
+nav_order: 1
+---
+# Install in AWS
+
+## AMI
+
+- Fyde Access Proxy AMI is based on the oficial Amazon Linux 2 AMI
+
+- The AMI is available in the account 766535289950 in all regions under the prefix `amazonlinux-2-base_*`
+
+- Fyde Access Proxy AMI details:
+  - [CIS](https://www.cisecurity.org/){:target="_blank"} recomendations for CentOS
+  - [CIS](https://www.cisecurity.org/){:target="_blank"} recomendations for SSH
```

Comment:
@GMartinez-Sisti is this supposed to take us to the main page?

Reply:
Yes, there is no specific link for SSH, so I sent them both to the main page, just to provide context on what CIS is.